aiida/orm/implementation/django/computer.py

```diff
@@ -222,6 +222,9 @@ def get_workdir(self):
 
     def set_workdir(self, val):
         # if self.to_be_stored:
+        if not isinstance(val, basestring):
```

Comment:
Consider using six.string_types (python 3 compatible), six is already an indirect dependency. May be too early to even think about that though

Reply:
I think this replacement should be practically automatic when we do the transition.
Let's do it when it becomes relevant.